app/views/support_interface/zendesk/export.csv.erb

```diff
@@ -0,0 +1,5 @@
+<%- headers = ['Ticket ID', 'Group Name', 'Received At', 'Closed At', 'Enquiry Type', 'No Action Required'] -%>
+<%= CSV.generate_line headers %>
+<%- @zendesk_delete_requests.each do |request| -%>
+  <%= CSV.generate_line([request.ticket_id, request.group_name, request.received_at, request.closed_at, request.enquiry_type, request.no_action_required]) -%>
+<%- end -%>
```

Comment:
This file has no trailing newline; can you configure your editor to always add them?

(Sorry, I don't nit about this usually, but it looks like our linters aren't configured to catch this.)